Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+//
+//  DetailngCodebaseViewController.swift
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+import SnapKit
```

Comment:
p3;

딱히 어딘가에 정리해둔건 없지만, 제가 MC2 하면서 나름 정리했던 컨벤션들이 있는데 노션에 추가해두었습니다 ! 확인 부탁드려요

```suggestion
import UIKit

import SnapKit
```
저희의 스타일론 import를 두 섹션으로 나눠서 사용하는데, 첫 번째 섹션은 First party 들이 들어가고, 두 번째 섹션엔 Third party들을 넣고 있어요 !
한 줄 띄워주시면 감사하겠습니다.

Reply:
> p3;
> 
> 딱히 어딘가에 정리해둔건 없지만, 제가 MC2 하면서 나름 정리했던 컨벤션들이 있는데 노션에 추가해두었습니다 ! 확인 부탁드려요
> 
> 저희의 스타일론 import를 두 섹션으로 나눠서 사용하는데, 첫 번째 섹션은 First party 들이 들어가고, 두 번째 섹션엔 Third party들을 넣고 있어요 ! 한 줄 띄워주시면 감사하겠습니다.

ef6da35 넵 감사합니다~